doc/source/guide/plotting.rst

```diff
@@ -210,12 +207,51 @@ The above example creates two side by side plots one with the overall view of th
 with a small area marked with a white box. That smaller view is then shown in the plot
 below it. The spacing between the two plots is controlled by fig.subplots_adjust().
 
-7. Specifying a Colormap
-------------------------
+7. Plotting Keywords
+--------------------
+
+As mentioned before for Map `~matplotlib.pyplot.imshow()` does most of the heavy
+lifting in the background while SunPy makes a number of choices for you so that
+you don't have to. Changing these defaults
+is made possible through some simple interfaces. Firstly you can pass any
+`~matplotlib.pyplot.imshow()` keyword into
+the plot command to override the defaults for that particular plot. The following example
+changes the default AIA color table to use an inverse Grey color table::
+
+    import sunpy.map
+    import sunpy.data.sample
+    import matplotlib.pyplot as plt
+    smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+    fig = plt.figure()
+    ax = plt.subplot(1,1,1)
+    smap.plot(cmap=plt.Greys_r)
+    plt.show()
+
+If you'd like to make this a permanent change you can access a number of settings under the
+`plot_settings` property to make your changes for that map instance permanent.
+In the following example we change the normalization of the color table to a linear
+one running from 5 to 100 (clipping everything above and below these values)::
+
+    import sunpy.map
+    import sunpy.data.sample
+    import matplotlib.colors as colors
+    smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+    smap.plot_settings['norm'] = colors.Normalize(5, 1000)
+
+    fig = plt.figure()
+    ax = plt.subplot(1,1,1)
+    smap.plot()
+    plt.show()
+
```

Comment:
Maybe something about masked arrays?
"SunPy map plotting respects numpy masked arrays.  For information on how to manipulate the colormap to show masked data, please consult http://matplotlib.org/examples/pylab_examples/contourf_demo.html"